core/controllers/classroom.py

```diff
@@ -32,7 +32,8 @@ class ClassroomPage(base.BaseHandler):
     def get(self):
         """Handles GET requests."""
 
-        if not constants.ENABLE_NEW_STRUCTURE_PLAYERS:
+        if not constants.ENABLE_NEW_STRUCTURE_PLAYERS or not (
+                constants.SHOW_CLASSROOM):
```

Comment:
@iamprayush, I wanted to know why this page shows an error when there isn't any data available? Why don't we show a message instead? Can you please explain?

Reply:
Well, since this page isn't "released" yet, isn't it appropriate to consider it as a non-existent page and throw a 404?